ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -183,29 +149,58 @@ private void OnOrientationChanged(DisplayInformation displayInformation, object
         private void SendUpdateDimensionsEvent()
         {
             Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
-                .emit("didUpdateDimensions", GetDimensions());
+                .emit("didUpdateDimensions", GetAllDimensions());
         }
 
-        private static JObject GetDimensions()
+        private JObject GetAllDimensions()
         {
-            return GetDimensions(DisplayMetrics.GetForCurrentView());
+            var dimensions = new JObject();
+
+            // Default metric for main window
+            // TODO It would make sense to make default actively focused window and not the main in the future
+            var mainView = _registeredViews.Values.FirstOrDefault(v => v.RootView.Dispatcher == CoreApplication.MainView.Dispatcher);
+            var defaultMetric = mainView == null ? DisplayMetrics.Empty : mainView.CurrentDisplayMetrics;
+            dimensions.Add("window", GetDimensions(defaultMetric));
+
+            foreach (var info in _registeredViews.Values)
+            {
+                dimensions.Add($"window-{info.RootViewTag}", GetDimensions(info.CurrentDisplayMetrics));
```

Comment:
>$"window-{info.RootViewTag}" [](start = 31, length = 28)

Instead of concatenating strings (and this having to do the same in JavaScript), can we just add another sub-object and have the key be the root tag only? E.g.:
{
   "1": { /* dimensions */ },
   "42": { /* alt-window dimensions */ }
}